packages/adapters-library/src/adapters/curve/tests/snapshots/ethereum.positions.position-2.json

```diff
@@ -84,6 +85,17 @@
           "balanceRaw": "6357359611959254406n",
           "type": "protocol",
           "tokens": [
+            {
+              "name": "Curve DAO Token",
+              "symbol": "CRV",
+              "decimals": 18,
+              "iconUrl": "https://raw.githubusercontent.com/trustwallet/assets/master/blockchains/ethereum/assets/0xD533a949740bb3306d119CC777fa900bA034cd52/logo.png",
+              "address": "0xD533a949740bb3306d119CC777fa900bA034cd52",
+              "balanceRaw": "97118831066726817n",
+              "type": "underlying-claimable",
+              "balance": 0.09711883106672682,
+              "price": 0.5718285537644536
+            },
```

Comment:
It's the same value, it's just been moved here because now we get rewards before unwrapping.